lib/Services/delorean.js

```diff
@@ -5,7 +5,14 @@ class Delorean {
    */
   constructor(url, Auth) {
     this.service = Delorean.name;
-    this.url = url;
+    // new env
+    if (url.indexOf('delorean-es') > -1) {
```

Comment:
Since this part is dependent on the URL being used for Delorean, let's instead create a new class `deloreanES.js`

```js
class DeloreanES {
  constructor(url, Auth) {
    this.service = DeloreanES.name
    this.url = url;
    this.auth = Auth;
  }

  getDeviceCheckIn
  getDeviceState
  getDevicesState
  getDeviceTotals
  getDeviceAggregations
  getDeviceReadings
  getDeviceReportReadings
}
```

SS-API will just need a new variable DELOREAN_ES_BASE_URL

Reply:
Separated es from old delorean. Will delete delorean url after switched all history to es version.